prime-router/src/main/kotlin/transport/RESTTransport.kt

```diff
@@ -252,25 +252,14 @@ class RESTTransport(private val httpClient: HttpClient? = null) : ITransport {
         var bearerTokens: BearerTokens? = null
         when (credential) {
             is UserApiKeyCredential -> {
-                bearerTokens = if (credential.user.equals("flexion")) {
-                    val tokenInfoEtor: TokenInfoEtor = getAuthTokenWithUserEtor(
-                        restTransportInfo.authTokenUrl,
-                        credential,
-                        logger,
-                        tokenClient
-                    )
-                    // if successful, add the token returned to the token storage
-                    BearerTokens(tokenInfoEtor.accessToken, "")
-                } else {
-                    tokenInfo = getAuthTokenWithUserApiKey(
-                        restTransportInfo.authTokenUrl,
-                        credential,
-                        logger,
-                        tokenClient
-                    )
-                    // if successful, add the token returned to the token storage
-                    BearerTokens(tokenInfo.accessToken, tokenInfo.refreshToken ?: "")
-                }
+                tokenInfo = getAuthTokenWithUserApiKey(
+                    restTransportInfo,
+                    credential,
+                    logger,
+                    tokenClient
+                )
+                // if successful, add the token returned to the token storage
+                bearerTokens =BearerTokens(tokenInfo.accessToken, tokenInfo.refreshToken ?: "")
```

Comment:
Did you run `./gradlew ktlintFormat`? Do you have your precommit checks set up? I wouldn't have been able to check this in because it doesn't conform to our code format restrictions, I _think_ those checks can be installed with `.environment/githooks.sh install`.
```suggestion
                bearerTokens = BearerTokens(tokenInfo.accessToken, tokenInfo.refreshToken ?: "")
```

Reply:
I will run the command, now.

Reply:
Done.